docs/installation/others/creating-ds-instance.adoc

```diff
@@ -26,28 +26,17 @@ _If you wish to disable bootstrap certificate generation and SSL connection, set
 $ dscreate create-template ds.inf
 ....
 
-Customize the DS configuration file as follows:
+Make sure to customize the resulting content of ds.inf to have the appropriate values. Specifically check (or set if needed) the following values:
 
-[literal,subs="+quotes,verbatim"]
-....
-$ sed -i \
-    -e "s/^;instance_name = .*/instance_name = localhost/" \
-    -e "s/^;root_password = .*/root_password = Secret.123/" \
-    -e "s/^;suffix = .*/suffix = dc=example,dc=com/" \
-    -e "s/^;create_suffix_entry = .*/create_suffix_entry = True/" \
-    -e "s/^;self_sign_cert = .*/self_sign_cert = True/" \
-    ds.inf
-....
-
-where
+* *instance_name* specifies the name of the DS instance. In this example let's set it to `localhost`.
```

Comment:
Previously "this example" was referring to the `sed` command. Since the command is no longer there "this example" might be confusing. Would it be better if we change it to: "For example, `localhost`."?

Reply:
@edewata Since other installation docs assume these example values (e.g. password, port numbers, etc.), how about we say "in our example ..." instead?  The statement above already tell them to set values to "appropriate values"

Reply:
Should we use "our" in the docs? Regardless, I'll defer to Doc team for the actual wording.